src/main/drivers/timer.h

```diff
@@ -50,6 +50,11 @@ typedef uint32_t timCCR_t;
 typedef uint32_t timCCER_t;
 typedef uint32_t timSR_t;
 typedef uint32_t timCNT_t;
+#elif defined(STM32H7)
```

Comment:
(eventually there may be one case for F1 (16 bit) and ano for remaining CPUs). This code was created when there was only F1 and F3 ...